opentelemetry-otlp/src/metric.rs

```diff
@@ -100,6 +100,56 @@ impl From<TonicExporterBuilder> for MetricsExporterBuilder {
     }
 }
 
+/// Configure delta temporality for all [InstrumentKind]
+///
+/// [Temporality::Delta] will be used for Counter, Asynchronous Counter and Histogram
+/// [Temporality::Cumulative] will be used for UpDownCounter and Asynchronous UpDownCounter
+#[derive(Clone, Default, Debug)]
+struct DeltaTemporalitySelector {
+    pub(crate) _private: (),
+}
+
+impl DeltaTemporalitySelector {
```

Comment:
We have the same thing defined in [benches](https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-sdk/benches/metric.rs#L73) currently. Is it worthwhile to consolidate this into a common place?